src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -330,9 +330,10 @@ def project_or_package_link(opts)
     defaults = { package: nil, rev: nil, short: false, trim_to: 40 }
     opts = defaults.merge(opts)
 
-    CacheLine.fetch(['project_or_package_link', opts], project: opts[:project], package: opts[:package]) do
-      # only care for database entries
-      prj = Project.where(name: opts[:project]).select(:id, :name).first
+    # only care for database entries
+    prj = Project.where(name: opts[:project]).select(:id, :name).first
```

Comment:
You are only selecting id and name from projects but you call `prj.cache_key` on line 336? Maybe just get rid of the `select` since I dont think there is really that much performance gain from only selecting the specific columns as opposed to selecting all of them?
```ruby
prj = Project.find_by(name: opts[:project])
```